frontend/uddhar-frontend/src/coordinator/components/DisasterInput.jsx

```
@@ -0,0 +1,190 @@
+  };
+
+
+  return (
+    <div className="flex items-center py-5 px-5 justify-center bg-gradient-to-b from-amber-200 to-gray-100 bg-opacity-50">
+      <div className="bg-white p-6 rounded-lg shadow-lg w-140">
+        <h2 className="text-xl font-bold mb-4">Provide Required Data</h2>
+
+        <form onSubmit={handleSubmit}>
+          {/* title of the disaster */}
+          <div className="mb-4">
+            <label className="block text-gray-700">Title</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)